onboarding/src/Components/hooks/Packages.js

```diff
@@ -1,15 +1,13 @@
-import React, { useState, useEffect } from "react";
+import { useState, useEffect } from "react";
```

Comment:
Removing this import causes e.g.:
`ESLint: React Hook "useState" is called in function "fetchPackages" which is neither a React function component or a custom React Hook function.(react-hooks/rules-of-hooks)`
`ESLint: React Hook "useEffect" is called in function "fetchPackages" which is neither a React function component or a custom React Hook function.(react-hooks/rules-of-hooks)`
This is underlined everywhere where `useEffect` or `useState` is called.

Reply:
I removed it, because JSX is not rendering in this file, so importing React here is not necessary.